Earthfile

```diff
@@ -47,9 +47,14 @@ generate:
 # e2e runs end-to-end tests. See test/e2e/README.md for details.
 e2e:
   ARG FLAGS="-test-suite=base"
-  # Docker installs faster on Alpine, and we only need Go for go tool test2json.
-  FROM golang:${GO_VERSION}-alpine3.20
-  RUN apk add --no-cache docker jq
+  # Using earthly image to allow comability with different development environments e.g. WSL
+  FROM earthly/dind
```

Comment:
is there a particular tag/version we should be using here too? i.e. what does `latest` point to?

From https://github.com/earthly/dind/blob/main/docs/dockerhub.md:

> Users, however, are encouraged to pin to specific version tags moving forward. The unversioned tags will be left as-is to help backwards-breaking changes. 


Also helpful best practices for `earthly/dind`: https://docs.earthly.dev/docs/guides/best-practices#use-earthly-dind

Reply:
Thanks for pointing that out! `latest` tag is pointing to an old image. I changed it to explicit Alpine and Docker version `alpine-3.20-docker-26.1.5-r0`, as with best practices.

Reply:
I wonder if this is something Renovate can keep up to date for us over time? 🤔